packages/mobile/src/screens/profile-screen/ProfileHeader/ProfileInfoTiles.tsx

```diff
@@ -268,11 +305,41 @@ export const ProfileInfoTiles = () => {
     'allow_ai_attribution'
   ])
 
+  const { isEnabled: isRecentCommentsEnabled } = useFeatureFlag(
+    FeatureFlags.RECENT_COMMENTS
+  )
+
   const accountId = useSelector(getUserId)
 
   const hasMutuals =
     user_id !== accountId && current_user_followee_follow_count > 0
 
+  const { data: recentComments = [], isLoading: loadingComments } =
+    useUserComments(user_id, 1)
+
+  // Only animate if comments are not immediately visible
+  const [useAnimation] = useState(loadingComments)
```

Comment:
@dylanjeffers Curious what you think about this pattern. The goal was to only enable animations if we haven't successfully loaded comments yet. Otherwise, we can just show the button. It's relying on the `useState` behavior where it only sets the value on the first render and the fact that the query above will return false for loading if it's already in the tan-query cache when this component mounts.

Reply:
Yeah I can't think of a better way to do this right now, makes sense